spring-cloud-dataflow-docs/src/main/asciidoc/configuration.adoc

```diff
@@ -89,11 +89,24 @@ You can secure your REST endpoints, as well as the Data Flow Dashboard by enabli
 and requiring clients to authenticate using either:
 
 * http://oauth.net/2/[OAuth 2.0]
-* Basic Authentication
+* Traditional Authentication (Incl. Basic Authentication)
+
+.Authentication Options
+image::{dataflow-asciidoc}/images/dataflow-authentication-options.png[Authentication Options, scaledwidth="80%"]
+
+When choosing _traditional authentication_, the Spring Cloud Data Flow server will
+be the main authentication point, using Spring Security under the covers. When
+selecting this option, user then need to further define their preferred authentication
+mechanism aka select the desired authentication backing store:
+
+* Single User Authentication
```

Comment:
Perhaps we could link each of the sub-chapters, so it would be easy to navigate.